src/components/transactions/TxDetails/TxData/Transfer/index.tsx

```diff
@@ -51,6 +54,7 @@ const TransferTxInfo = ({ txInfo, txStatus, trusted }: TransferTxInfoProps & { t
           <TransferActions address={address.value} txInfo={txInfo} trusted={trusted} />
         </EthHashInfo>
```

Comment:
@jmealy 
I know this is already closed but the trusted flag is used for address poisoning as well here.

I think we should pass the new imitation flag also into the `EthHashInfo` so we get the "Before you copy" modal if someone tries to copy the address.
```ts
        <EthHashInfo
          address={address.value}
          name={address.name}
          customAvatar={address.logoUri}
          shortAddress={false}
          hasExplorer
          showCopyButton
          trusted={trusted || imitation}
        >
 ```

Reply:
Good spot, thanks!